addons/barcodes_gs1_nomenclature/data/barcodes_gs1_rules.xml

```diff
@@ -334,5 +334,797 @@
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3120</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(31[23][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_meter"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+        </record>
+
+        <record id="barcode_rule_gs1_324y_325y_326y_327y_328y_329y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3240</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(32[4-9][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+        </record>
+
+        <record id="barcode_rule_gs1_from_330y_to_337y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3300</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(33[0-7][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+        </record>
+
+        <record id="barcode_rule_gs1_from_340y_to_349y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3400</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(34[0-9][0-5])(\d{6})</field>
+            <field name="associated_uom_id" ref="uom.product_uom_unit"/>
+            <field name="type">skip</field>
+            <field name="gs1_content_type">measure</field>
+            <field name="gs1_decimal_usage">True</field>
+        </record>
+
+        <record id="barcode_rule_gs1_350y_and_from_352y_to_356y" model="barcode.rule">
+            <field name="name">Unused GS1 AI measure rules</field>
+            <field name="barcode_nomenclature_id" ref="default_gs1_nomenclature"/>
+            <field name="sequence">3500</field>
+            <field name="encoding">gs1-128</field>
+            <field name="pattern">(35[02-6][0-5])(\d{6})</field>
```

Comment:
split into 6 (i.e., yay for the imperial system of units, continued).

- Area, square inches
- Area, square yards (variable measure trade item)
- Area, square inches
- Area, square feet
- Area, square yards
- Net weight, troy ounces (variable measure trade item) <= had to look up this one, it's a weight UoM that we obviously don't have (apparently it's used for precious metals apparently)